browser/css/jssidebar.css

```diff
@@ -89,14 +94,9 @@
 .sidebar.ui-expander {
 	display: flex;
 	justify-content: space-between;
-	width: calc(100% - 10px);
 	align-items: center;
 }
 
-.sidebar.ui-expander-content {
-	width: calc(100% - 10px);
-}
-
```

Comment:
:question: Maybe this fixes or improves something else? If so please add as a different commit with respective commit message

Reply:
this is needed as a placeholder for scrollbar, are you sure when we resize window - sidebar will not cover any elements without that code?